### PR TITLE
Feature/kids 285 edit avatar

### PR DESCRIPTION
**Pull request by pavelpavelko**
## Description
- Fetching avatars logic is implemented;
- Avatars selection screen is implemented; 
- Edit child profile feature is implemented;
- Home screen is slightly updated with the new design;

### CODE REVIEW

File: lib/core/network/api_service.dart [hunk 1]
```diff
@@ -234,4 +234,51 @@ class APIService {
       return itemsList;
     }
   }
+
+  Future<List<dynamic>> fetchAvatars() async {
+    final url = Uri.https(_apiURL, '/givt4kidsservice/v1/profiles/avatars');
+
+    final response = await client.get(url);
+
+    log('fetch avatars response code: ${response.statusCode}');
```
Comment (MaikelStuivenberg) [inline lib/core/network/api_service.dart:243]: I would remove the debug logging from the code. (its getting logged to logit in dev and prod)

File: lib/core/network/api_service.dart [hunk 2]
```diff
@@ -234,4 +234,51 @@ class APIService {
...
+    var response = await client.put(
+      url,
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+      },
+      body: jsonEncode(body),
+    );
+
+    log('edit-profile status code: ${response.statusCode}');
```
Comment (MaikelStuivenberg) [inline lib/core/network/api_service.dart:273]: same here

File: lib/features/edit_profile/cubit/edit_profile_cubit.dart [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          currentProfilePicture:
+              _extractFileNameFromPictureUrl(currentProfilePicture),
+          selectedProfilePicture:
+              _extractFileNameFromPictureUrl(currentProfilePicture),
+        ));
+
+  final EditProfileRepository editProfileRepository;
+  final String childGUID;
+
+  static String _extractFileNameFromPictureUrl(String url) {
```
Comment (MaikelStuivenberg) [inline lib/features/edit_profile/cubit/edit_profile_cubit.dart:23]: It is simple solution, but we can also make use of existing packages by dart to do things with paths 

https://pub.dev/packages/path
https://pub.dev/documentation/path/latest/path/basename.html

File: lib/features/profiles/widgets/wallet_widget.dart [hunk 4]
```diff
@@ -1,21 +1,42 @@
...
+  @override
+  State<WalletWidget> createState() => _WalletWidgetState();
+}
+
+class _WalletWidgetState extends State<WalletWidget> {
+  Future<String> _getAppIDAndVersion() async {
+    final packageInfo = await PackageInfo.fromPlatform();
+    final result =
+        '${packageInfo.packageName} v${packageInfo.version}(${packageInfo.buildNumber})';
+    log(result);
```
Comment (MaikelStuivenberg) [inline lib/features/profiles/widgets/wallet_widget.dart:36]: remove debug logging?

**Review by MaikelStuivenberg — APPROVED** [review]

File: lib/features/edit_profile/cubit/edit_profile_cubit.dart (hunk 3 above)
Comment (pavelpavelko) [reply lib/features/edit_profile/cubit/edit_profile_cubit.dart:26]: re-implemented with path package, thanks)

**Pull request merged**